### DISCUSSION TITLE
Rules Channels, and other channels that require permanent messages.

### DISCUSSION THREAD

**Discussion by MagmaSlasher** [Valour-Software/Valour#85] [Q&A] [answered]
So, messages are deleted after 24h. Totally understandable, messages aren’t necessary to be kept permanently, so there’s no need to keep them. But, how will rules channels work, amongst other things? Some channels *will* need permanent messages. Whether that be rules, templates, etc. How will these work?

(Sorry if I rambled a bit-)

**Comment by fruitcake5** [comment, marked as answer]
Messages are deleted from the Valour servers after 24 hours, the proxied peer to peer system will allow online clients to serve the messages
